notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb

```diff
@@ -0,0 +1,398 @@
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.6/dist-packages (4.6.3)\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "iU2jp_nSS-Nv",
+        "outputId": "0069d10d-9f14-4881-c1f8-fcb0d183883e"
+      },
+      "source": [
+        "# parser(s)\r\n",
+        "!pip install lxml \r\n",
+        "  # prefered for use with bs4\r\n",
+        "!pip install html5lib \r\n",
+        "  # popular parser that will also work with bs4"
+      ],
+      "execution_count": 13,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Requirement already satisfied: lxml in /usr/local/lib/python3.6/dist-packages (4.2.6)\n",
+            "Requirement already satisfied: html5lib in /usr/local/lib/python3.6/dist-packages (1.0.1)\n",
+            "Requirement already satisfied: six>=1.9 in /usr/local/lib/python3.6/dist-packages (from html5lib) (1.15.0)\n",
+            "Requirement already satisfied: webencodings in /usr/local/lib/python3.6/dist-packages (from html5lib) (0.5.1)\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "YAM1GOlnTAyZ",
+        "outputId": "e1bcc1c7-343d-4fc7-e66c-e5d33acd8b28"
+      },
+      "source": [
+        "!pip install requests"
+      ],
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Requirement already satisfied: requests in /usr/local/lib/python3.6/dist-packages (2.23.0)\n",
+            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests) (3.0.4)\n",
+            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests) (2020.12.5)\n",
+            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests) (2.10)\n",
+            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests) (1.24.3)\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Bg2VHnbvbSTi"
+      },
+      "source": [
+        "from bs4 import BeautifulSoup as bs\r\n",
+        "import requests\r\n",
+        "import csv\r\n"
+      ],
+      "execution_count": 1,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PfsINlO7bhwZ"
+      },
+      "source": [
```

Comment:
Could you insert line breaks here so that the full text is readable?

Reply:
This is markdown and auto-formats to the size of the screen when viewing in the notebook.  Unfortunately, when reviewing in GitHub, we have to scroll.